### PR TITLE
🐛 Changes use of Map to an Object, fixing IE polyfill bug.

### PR DESCRIPTION
**Pull request by aendrew**
Currently `Map` causes IE to chuck a wobbly. I've replaced my use of it with an `Object`.

### CODE REVIEW

**Comment by JakeChampion** [comment]
LGTM 👍

File: client/components/us-election-headpiece/index.js [hunk 1]
```diff
@@ -5,28 +5,36 @@ import { initSpinner } from './progress-indicator';
...
-	breakpoints.set('L', 980);  // column-width: 60px, inner width: 940px
-	breakpoints.set('XL',1220); // column-width: 80px, inner width: 1180px
-
-	breakpoints.forEach((width, size) => {
+	// const breakpoints = new Map();
+	// breakpoints.set('default', 489);
+	// breakpoints.set('S', 490);  // column-width: 30px, inner width: 470px
+	// breakpoints.set('M', 740);  // column-width: 40px, inner width: 700px
+	// breakpoints.set('L', 980);  // column-width: 60px, inner width: 940px
+	// breakpoints.set('XL',1220); // column-width: 80px, inner width: 1180px
```
Comment (leggsimon) [inline client/components/us-election-headpiece/index.js]: Could we remove these comments please

Comment (aendrew) [inline client/components/us-election-headpiece/index.js]: @leggsimon I'll be submitting a new PR for a bunch of changes I'm making to the headpiece later today — will remove them in that. Is that sufficient?

Comment (leggsimon) [inline client/components/us-election-headpiece/index.js]: Great yeah thanks 😺

**Pull request merged**